Makefile

```diff
@@ -43,6 +43,8 @@ fix:
 
 fmt:
 	$(PYTHON) -m ruff format .
+	$(PYTHON) -m autoflake --remove-all-unused-imports --in-place --recursive .
+	$(PYTHON) -m isort .
```

Comment:
If you're using `ruff`, you can sort imports: https://docs.astral.sh/ruff/formatter/#sorting-imports